service/marathon.json.mustache

```diff
@@ -53,7 +53,12 @@
   "container": {
     "type": "DOCKER",
     "docker": {
+{{#scala-2-11}}
```

Comment:
These changes need to be put into https://github.com/mesosphere/universe/tree/version-3.x/repo/packages/F/flink/0
I will take care of this.